Makefile

```diff
@@ -292,7 +292,7 @@ schemas_test:
 	(cd schemas && poetry run pytest)
 
 schemas_check: schemas_install schemas_black schemas_ruff schemas_test
-	(cd schemas && poetry run pydantic2ts --module mozilla_nimbus_schemas.jetstream --output /tmp/test_index.d.ts --json2ts-cmd "yarn json2ts")
+	(cd schemas && poetry run pydantic2ts --module mozilla_nimbus_schemas.__init__ --output /tmp/test_index.d.ts --json2ts-cmd "yarn json2ts")
```

Comment:
Nice, obvious in retrospect! Whoops. Can you just pass `mozilla_nimbus_schemas` as the module now that there's an `__init__` file?

Reply:
Sadly no becuase `mozilla_nimbus_schemas` is both the name of the folder and the name of the Python import path, and [here](https://github.com/phillipdupuis/pydantic-to-typescript/blob/master/pydantic2ts/cli/script.py#L34-L42) they check whether it's a valid path FIRST and try to do the path loading mechanism which isn't working for whatever reason, so explicitly pointing it to `mozilla_nimbus_schema.__init__` makes it not a valid file path but a valid Python import path so it forces the check into the Python import flow which does work.  Silly nuance of the module.

Reply:
Ah, oh well. At least this catches everything now. Thanks!